native/cocos/network/WebSocket-libwebsockets.cpp

```diff
@@ -234,7 +234,7 @@ class WebSocketImpl {
     int onClientReceivedData(void *in, ssize_t len);
     int onConnectionOpened();
     int onConnectionError();
-    int onConnectionClosed();
+    int onConnectionClosed(uint16_t code, const ccstd::string& reason);
```

Comment:
Why not use `int` for parameter code?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/CloseEvent said that `code` is an `unsigned short`